ft_databrowser.m

```diff
@@ -2060,8 +2070,22 @@ function redraw_cb(h, eventdata)
       
       % laychan is the actual topo layout, in pixel units for .mat files
       % laytopo is a vertical layout determining where to plot each topo, with one entry per component
+     
+     opt = {'interpmethod', cfg.interpolation, ... 
```

Comment:
this introduces a bug in the code, since `opt` was already used as a variable name (and was a struct rather than a cell-array)

Reply:
poor variables called 'opt'...